src/main/java/com/sumologic/jenkins/jenkinssumologicplugin/sender/LogSender.java

```diff
@@ -50,29 +57,29 @@ public static LogSender getInstance() {
         return LogSenderHolder.logSender;
     }
 
-    void sendLogs(String url, byte[] msg, String sumoName, String sumoCategory, String contentType) {
+    void sendLogs(String url, byte[] msg, String sumoName, String sumoCategory, String contentType, HashMap<String, String> fields, String host) {
```

Comment:
Not backward compatible, is this function just internal?

Reply:
Function is internal and used by sumoupload function